### DISCUSSION TITLE
How to implement MaterialTopTabs

### DISCUSSION THREAD

**Discussion by topzdev** [expo/router#571] [Q&A]
There is no documentation on how to implement matarialTopTabs here in the expo router. 

https://reactnavigation.org/docs/material-top-tab-navigator

How to implement it? 
THanks

**Comment by likeSo** [comment]
See https://github.com/EvanBacon/expo-router-layouts-example
BTW, I think these layouts should be expo-router built-in layouts too.

**Comment by BoavistaLudwig** [comment]
> There is no documentation on how to implement matarialTopTabs here in the expo router.
> 
> https://reactnavigation.org/docs/material-top-tab-navigator
> 
> How to implement it? THanks

Did you implement it? If so, did you encounter any issues with rendering of the last screen of the Material Top Tab Navigator?
